python/lsst/ts/scheduler/model.py

```diff
@@ -45,9 +46,16 @@
 from .driver.driver_target import DriverTarget
 from .driver.survey_topology import SurveyTopology
 from .telemetry_stream_handler import TelemetryStreamHandler
-from .utils.csc_utils import FailedStates, NonFinalStates, is_uri, is_valid_efd_query
+from .utils.csc_utils import (
+    BlockStatus,
+    FailedStates,
+    NonFinalStates,
```

Comment:
It looks like `FailedStates` and `NonFinalStates` were defined before this PR but I'd like to point out that our naming convention dictates that single nouns are preferred, so `FailedState` and `NonFinalState`.

Reply:
I would agree if they were classes, but these are constants with a frozen set of values.